swim/memberlist.go

```diff
@@ -79,11 +84,21 @@ func (m *memberlist) ComputeChecksum() {
 	startTime := time.Now()
 	m.members.Lock()
 	checksum := farm.Fingerprint32([]byte(m.GenChecksumString()))
+	oldChecksum := m.members.checksum
 	m.members.checksum = checksum
 	m.members.Unlock()
+
+	if oldChecksum != checksum {
```

Comment:
Turns out the branch is covered during testing: https://coveralls.io/builds/5201500/source?filename=swim%2Fmemberlist.go#L91